tests/unit_tests/test_game.py

```diff
@@ -220,9 +220,39 @@ def custom_game(event_loop, database, game_service, game_stats_service):
     mock_game_connection.player = players.hosting
     mock_game_connection.state = GameConnectionState.CONNECTED_TO_HOST
     game.add_game_connection(mock_game_connection)
+    # Players should not be considered as 'in lobby' until the host has sent
+    # "PlayerOption" configuration for them
+    assert game.to_dict()["num_players"] == 0
```

Comment:
Maybe also add
```
assert len(game.players) == 0
```
Even if that's called by `to_dict` now it might not at some point in the future.

Reply:
I was trying to avoid making it too implementation specific, but maybe it's a good idea.